core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/GlobalDefinitionsIT.java

```diff
@@ -159,6 +158,18 @@ public void testCamelBeanPropertiesSpring() throws IOException, CoreException {
 		assertThat(bean3prop1.getAttribute(GlobalBeanEIP.PROP_NAME)).isEqualTo("description");
 		assertThat(bean3prop1.getAttribute(GlobalBeanEIP.PROP_VALUE)).isEqualTo("#1 account");
 		
+		CamelBean beanWithAllAttributes = (CamelBean)model1.getGlobalDefinitions().get("whateverWithAllFields");
+		checkAllCommonAttributesLoaded(beanWithAllAttributes);
+		assertThat(beanWithAllAttributes.getParameter(GlobalBeanEIP.PROP_FACTORY_BEAN)).isEqualTo("factoryMethod");
+	}
+
+	private void checkAllCommonAttributesLoaded(CamelBean beanWithAllAttributes) {
```

Comment:
What do you want to split more?
In this method there is all common attributes in Spring and Blueprint and the different attribute is checked in 2 different test method.
Do you mean that it would be better to have 2 distinct test classes?